src/main/webapp/index.html

```diff
@@ -183,6 +184,18 @@ <h1><a href="/">Census Viz</a></h1>
       <a id="census-link" target="_blank" rel="noopener">Explore more data from the Census API</a>
       <br>
       <div id="data-table"></div>
+      <div id="yearly-total-chart-loading-msg"></div>
+      <div id="yearly-total-chart" class="yearly-chart"></div>
+      <div id="yearly-county-chart-loading-msg"></div>
+      <form id="county-form" onsubmit="createYearlyCountyChart(); return false;">
+        <label for="county-list">View yearly data for a county</label>
```

Comment:
What would you think of phrasing this as "View yearly data for" ? Fitting with our main query that uses the dropdown as kind of part of the sentence.
Alternatively, if that's too weird, I think it'd be nice to add a colon ("...for a county:") to separate the text and the label a bit more.

Reply:
Switched to a question format to try to match main query: "how many {personType} {action} in {county} each year?", let me know your thoughts on this or whether you prefer "yearly data for"

Reply:
I like that! Thanks.